app/src/lib/stores/repositories-store.ts

```diff
@@ -289,7 +299,12 @@ export class RepositoriesStore extends BaseStore {
       )
     }
 
-    return record!.lastStashCheckDate
+    lastCheckDate = record!.lastStashCheckDate
```

Comment:
feel free to punt on this to the PR for `baseline-stashing-metrics2`, but why do we need to `!` here?

Reply:
It's the TS way of saying "I know this value might be null/undefined and I don't care, let's use it anyway".

We try to avoid these unless absolutely necessary, and I think we should be able to drop this because of the previous `if (record === undefined)`

Reply:
Nope!